src/scene/gsplat/gsplat-sogs-data.js

```diff
@@ -310,6 +310,7 @@ class GSplatSogsData {
         const { device } = means_l;
         const { scope } = device;
 
+        // Note: do not destroy it, keep it available for the lifetime of the app
         const shader = ShaderUtils.createShader(device, {
```

Comment:
the variable should be stored globally then?

Reply:
It's in the cache using provided unique name, no need to handle it here too. This matches other shaders used internally in the engine

Reply:
ok, the name `createShader` is not the best then, because it seems to be equivalent to `new Shader`.